otherlibs/win32unix/utimes.c

```diff
@@ -0,0 +1,82 @@
+/**************************************************************************/
+/*                                                                        */
+/*                                 OCaml                                  */
+/*                                                                        */
+/*             Xavier Leroy, projet Cristal, INRIA Rocquencourt           */
+/*                                                                        */
+/*   Copyright 2017 Institut National de Recherche en Informatique et     */
+/*     en Automatique.                                                    */
+/*                                                                        */
+/*   All rights reserved.  This file is distributed under the terms of    */
+/*   the GNU Lesser General Public License version 2.1, with the          */
+/*   special exception on linking described in the file LICENSE.          */
+/*                                                                        */
+/**************************************************************************/
+
+#define CAML_INTERNALS
+
+#include <caml/fail.h>
+#include <caml/mlvalues.h>
+#include <caml/memory.h>
+#include <caml/signals.h>
+#include <caml/osdeps.h>
+#include "unixsupport.h"
+
+#include <windows.h>
+
+static void convert_time(double unixTime, FILETIME* ft)
```

Comment:
This is a duplicate (or at least closely related) to the same function in stat.c - this can be factored into unixsupport.c instead, I think?

Reply:
The function in `stat.c` does a `FILETIME -> double` conversion, while here we use its inverse `double -> FILETIME`.

Reply:
Yes, indeed - in which case, could they both go into unixsupport.c, but with different names?

Reply:
Yes, then can! But I would rather keep that change out of the present PR just for simplicity.